release_tools/semverup.py

```diff
@@ -149,6 +168,25 @@ def read_version_number(filepath):
     return version
 
 
+def bump_version_number(current_version, major=False, minor=False, patch=False):
```

Comment:
Maybe a better name for this function would be `force_new_version_number`, so it makes clear what the difference with `determine_new_version_number` is.